src/dex/fluid-dex/fluid-dex-pool.ts

```diff
@@ -50,6 +47,15 @@ export class FluidDexEventPool extends StatefulEventSubscriber<FluidDexPoolState
     this.handlers['LogOperate'] = this.handleOperate.bind(this);
   }
 
+  private padHexTo32Bytes(hex: string): string {
```

Comment:
such things should not be necessary. also decoding usually works via ABI and interface. Feel like there should be better solutions than manually parsing from raw

https://docs.ethers.org/v5/api/utils/bytes/#utils-hexZeroPad

Reply:
removed this function